Rubberduck.Parsing/ParserRuleContextExtensions.cs

```diff
@@ -254,6 +254,29 @@ public static bool ContainsTokenIndex(this ParserRuleContext context, int tokenI
             return precedingContext != null;
         }
 
+        /// <summary>
+        /// Returns the context containing the token following the context provided it is of the specified generic type.
```

Comment:
I have trouble parsing this documentation. It talks about a "following" context, but the out parameter is named `precedingContext`.

Also this sentence is rather long.